operator_ui/src/components/JobRuns/StatusCard.tsx

```diff
@@ -36,17 +38,56 @@ const styles = (theme: Theme) =>
     },
     statusText: {
       display: 'inline-block',
-      paddingLeft: theme.spacing.unit * 2,
-      textTransform: 'capitalize'
+      textTransform: 'capitalize',
+      color: '#3d5170'
```

Comment:
This should use theme customization https://material-ui.com/guides/typescript/#customization-of-theme